ultravox/tools/push_to_hub.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+import dataclasses
+from typing import Optional
+
+import simple_parsing
+
+from ultravox.inference import ultravox_infer
+
+
```

Comment:
Suggest adding a comment with an example command line, as in the PR description

Reply:
All done.